test/files/base64.iced

```diff
@@ -19,6 +19,6 @@ exports.test_b64_compat = (T,cb) ->
     for padlen in [0...pad.length]
       buf = Buffer.concat [ pad[0...padlen], x[0...i] ]
       e1 = b64stripped.encode buf
-      e2 = b64enc.encode buf
+      e2 = b64enc.encode buf, true
```

Comment:
Standalone boolean flags always make me worry that it'll be hard for someone reading the code here to tell what they mean. Maybe we could put this argument inside an `opts` dictionary, so that its name is visible here as `{old_shift: true}` or something like that.